mix.exs

```diff
@@ -34,12 +34,12 @@ defmodule ExJsonSchema.Mixfile do
   defp deps do
     [
       {:decimal, "~> 2.0"},
-      {:dialyxir, "~> 0.5", only: [:test], runtime: false},
-      {:ex_doc, ">= 0.0.0", only: :dev, runtime: false},
-      {:excoveralls, "~> 0.10", only: :test},
-      {:httpoison, "~> 0.8", only: :test},
-      {:mix_test_watch, "~> 0.2.6", only: [:dev, :test]},
-      {:poison, "~> 1.5", only: :test}
+      {:dialyxir, "~> 1.2", only: [:test], runtime: false},
+      {:ex_doc, ">= 0.28.5", only: :dev, runtime: false},
```

Comment:
Any reason why you included the patch version in the constraint? Since all of the deps are dev/test deps, I think it's fine to be a bit looser and allow minor version updates.

Reply:
I have loosened some of the constraints.